### PR TITLE
Week2 homework

### PR DESCRIPTION
**Pull request by BassamHajar**
thanks & regards,,,

### CODE REVIEW

File: week2/homework/src/index.js [hunk 1]
```diff
@@ -1,3 +1,34 @@
 'use strict';
 
-// TODO: Write the homework code in this file
+const commandList = require('./commandsList');
+const theMethods = require('./methods.js');
+
+switch (commandList[0]) {
```
Comment (saragerion) [inline week2/homework/src/index.js]: Remember that we would like to to display the `help` list also running `node index.js` (no parameters).
Tip: check if `commandList[0]` is empty :)

File: week2/homework/src/index.js [hunk 2]
```diff
@@ -1,3 +1,34 @@
 'use strict';
 
-// TODO: Write the homework code in this file
+const commandList = require('./commandsList');
+const theMethods = require('./methods.js');
```
Comment (saragerion) [inline week2/homework/src/index.js]: Not an "error", but generally variable names do not contain articles ("the", "a")

Comment (saragerion) [reply week2/homework/src/index.js]: Just "methods" would look great!

File: week2/homework/src/methods.js [hunk 3]
```diff
@@ -0,0 +1,94 @@
+const fs = require('fs');
+const util = require('util');
+
+const readFile = util.promisify(fs.readFile);
+const writeFile = util.promisify(fs.writeFile);
```
Comment (saragerion) [inline week2/homework/src/methods.js]: This variable is declared, but it is actually never used in this file. Tip: check the difference between how you use `readFile` and `writeFile`.

File: week2/homework/src/methods.js [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+    });
+  },
+
+  // DELETE:
+  delete: id => {
+    readFile(__dirname + '/todo.txt', 'utf-8').then(content => {
+      const contentArr = content.split('\n');
+      contentArr.forEach((item, index) => {
+        if (id == index + 1) {
+          const removed = contentArr.splice(id - 1, 1);
```
Comment (saragerion) [inline week2/homework/src/methods.js]: `removed` is never used. Not an "error", but it's always good to remember to remove unused code at the end.
It makes the code neater, more elegant and readable 👍

File: week2/homework/src/index.js (hunk 1 above)
Comment (BassamHajar) [reply week2/homework/src/index.js]: implemented as requested.

File: week2/homework/src/index.js (hunk 2 above)
Comment (BassamHajar) [reply week2/homework/src/index.js]: done.

File: week2/homework/src/methods.js (hunk 3 above)
Comment (BassamHajar) [reply week2/homework/src/methods.js]: done.

File: week2/homework/src/methods.js (hunk 4 above)
Comment (BassamHajar) [reply week2/homework/src/methods.js]: done.

**Pull request closed without merging**